src/app/app.component.html

```diff
@@ -1,16 +1,14 @@
-<td-layout>
-  <div td-toolbar-content>
+<md-sidenav-container fullscreen>
```

Comment:
I assume it's a temporary layout. The one we want is this: https://teradata.github.io/covalent/#/layouts/nav-list